Plugin.pm

```diff
@@ -1800,4 +1837,87 @@ sub _localDate {  # convert input date string in format YYYY-MM-DD to localized
 	return strftime(preferences('server')->get('shortdateFormat'), 0, 0, 0, $dt[2], $dt[1] - 1, $dt[0] - 1900);
 }
 
+sub addRecentSearch {
+	my $search = shift;
+	main::DEBUGLOG && $log->is_debug && $log->debug("++addRecentSearch");
+
+	my $list = $prefs->get('qobuz_recent_search') || [];
+
+	$list = [ grep { $_ ne $search } @$list ];
+
+	push @$list, $search;
+
+	shift(@$list) while scalar @$list > MAX_RECENT;
```

Comment:
`splice(@$list, -1 * MAX_RECENT - 1)`?

Reply:
I think `shift(@list) while...` reads better. See also reply above.

PR updated with the strings.txt change.

Reply:
Fair enough!